dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/SfdrData.kt

```diff
@@ -0,0 +1,230 @@
+
+    val averageGrossHourlyEarningsMaleEmployees: DataPoint<BigDecimal>?,
+
+    val averageGrossHourlyEarningsFemaleEmployees: DataPoint<BigDecimal>?,
+
+    val femaleBoardMembers: DataPoint<BigDecimal>?,
+
+    val maleBoardMembers: DataPoint<BigDecimal>?,
+
+    val controversialWeaponsExposure: DataPoint<YesNo>?,
+
+    val inorganicPollutants: DataPoint<BigDecimal>?,
+
+    val airPollutants: DataPoint<BigDecimal>?,
+
+    val ozoneDepletionSubstances: DataPoint<BigDecimal>?,
+
+    val carbonReductionInitiatives: DataPoint<YesNo>?,
+
+    val nonRenewableEnergyConsumptionFossilFuels: DataPoint<BigDecimal>?,
+
+    val nonRenewableEnergyConsumptionCrudeOil: DataPoint<BigDecimal>?,
+
+    val nonRenewableEnergyConsumptionNaturalGas: DataPoint<BigDecimal>?,
+
+    val nonRenewableEnergyConsumptionLignite: DataPoint<BigDecimal>?,
+
+    val nonRenewableEnergyConsumptionCoal: DataPoint<BigDecimal>?,
+
+    val nonRenewableEnergyConsumptionNuclearEnergy: DataPoint<BigDecimal>?,
+
+    val nonRenewableEnergyConsumptionOther: DataPoint<BigDecimal>?,
+
+    val waterConsumption: DataPoint<BigDecimal>?,
+
+    val waterReused: DataPoint<BigDecimal>?,
+
+    val waterManagementPolicy: DataPoint<YesNo>?,
+
+    val waterStressAreaExposure: DataPoint<YesNo>?,
+
+    val manufactureOfAgrochemicalPesticidesProducts: DataPoint<YesNo>?,
+
+    val landDegradationDesertificationSoilSealingExposure: DataPoint<YesNo>?,
+
+    val sustainableAgriculturePolicy: DataPoint<YesNo>?,
+
+    val sustainableOceansAndSeasPolicy: DataPoint<YesNo>?,
+
+    val wasteNonRecycled: DataPoint<BigDecimal>?,
+
+    val threatenedSpeciesExposure: DataPoint<YesNo>?,
+
+    val biodiversityProtectionPolicy: DataPoint<YesNo>?,
+
+    val deforestationPolicy: DataPoint<YesNo>?,
+
+    val securitiesNotCertifiedAsGreen: DataPoint<YesNo>?,
+
+    val workplaceAccidentPreventionPolicy: DataPoint<YesNo>?,
+
+    val rateOfAccidents: DataPoint<BigDecimal>?,
+
+    val workdaysLost: DataPoint<BigDecimal>?,
+
+    val supplierCodeOfConduct: DataPoint<YesNo>?,
+
+    val grievanceHandlingMechanism: DataPoint<YesNo>?,
+
+    val whistleblowerProtectionPolicy: DataPoint<YesNo>?,
+
+    val reportedIncidentsOfDiscrimination: DataPoint<BigDecimal>?,
+
+    val sanctionsIncidentsOfDiscrimination: DataPoint<BigDecimal>?,
+
+    val ceoToEmployeePayGap: DataPoint<BigDecimal>?,
+
+    val humanRightsDueDiligence: DataPoint<YesNo>?,
+
+    val traffickingInHumanBeingsPolicy: DataPoint<YesNo>?,
+
+    val reportedChildLabourIncidents: DataPoint<YesNo>?,
+
+    val reportedForcedOrCompulsoryLabourIncidents: DataPoint<YesNo>?,
+
+    val reportedIncidentsOfHumanRights: DataPoint<BigDecimal>?,
+
+    val reportedCasesOfBriberyCorruption: DataPoint<YesNo>?,
+
+    val reportedConvictionsOfBriberyCorruption: DataPoint<BigDecimal>?,
+
+    val reportedFinesOfBriberyCorruption: DataPoint<BigDecimal>?,
+
+    val referencedReports: Map<String, CompanyReport>? = null,
```

Comment:
The refenceReports are required to upload a datapoint which consists not only of the value but also of the quality bucket the report the number comes from and so on. And the reports referenced there can only be one of the referencedReports. Hence LkSG does not need this field since it simply does not use the datapoint class but self reported single field values without a source.